ert_shared/dark_storage/endpoints/updates.py

```diff
@@ -24,3 +24,13 @@ def get_update(
     update_id: UUID,
 ) -> js.UpdateOut:
     raise NotImplementedError
+
+
+@router.post("/updates/facade")
+def refresh_facade(
+    *,
+    res: LibresFacade = Depends(reset_res),
+) -> str:
+    if res is not None:
+        raise ValueError("Could not clean the ert facade")
+    return "Done"
```

Comment:
Does this return a OK response ?

Reply:
You are right, returning anything here makes no sense! :+1: